src/window.rs

```diff
@@ -1126,15 +1126,15 @@ impl Default for CursorIcon {
 }
 
 /// Fullscreen modes.
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Clone, Debug, PartialEq, Eq)]
```

Comment:
Nit: Add changelog entry

Reply:
I don't think internal impl requires it? Also I'm not sure that we should add changelog notes for `Eq` impls, since they are not breaking the code and transparent.

Reply:
Internal impls don't matter.

It's not a breaking change, but in general I think all API changes should be documented in the changelog, including small ones like this. But I don't have a strong opinion on this, you're free to disagree.

Reply:
A changelog entry would indeed be nice.